applications/OptimizationApplication/tests/test_execution_policies.py

```diff
@@ -0,0 +1,32 @@
+
+import KratosMultiphysics as Kratos
+
+# Import KratosUnittest
+import KratosMultiphysics.KratosUnittest as kratos_unittest
+
+from KratosMultiphysics.OptimizationApplication.execution_policies.execution_policy_wrapper import ExecutionPolicyWrapper
+
+class TestExecutionPolicies(kratos_unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.model = Kratos.Model()
+
+    def test_IndependentAnalysisExecutionPolicy(self):
+        parameters = Kratos.Parameters("""{
+            "name"                     : "test",
+            "execution_policy_type"    : "KratosMultiphysics.OptimizationApplication.execution_policies.independent_analysis_execution_policy.IndependentAnalysisExecutionPolicy",
```

Comment:
@sunethwarna can't we reduce/simplify the execution type ?! the input is long and might be confusing. As far as I understood, we have stepping and independent policies and why not to only say Stepping or Independent ?!
One more thing, let's set the stepping as the default type or ?!

Reply:
The reasoning is, if we do it like this, users can have their own `ExecutionPolicy` made in the folder they have the optimization case for custom execution (like coupling with different solvers, OpenFOAM, etc to run some types of analysis). I can make it shorter by putting `module_name`, and `class_name` into seperate json strings.

Reply:
Would it be ok, if I shorten it but still keeping the flexibility?

Reply:
Sure, it would be great.